src/include/miopen/mha/problem_description.hpp

```diff
@@ -51,6 +51,18 @@ struct MIOPEN_INTERNALS_EXPORT ProblemDescription : ProblemDescriptionBase
     }
 
     bool IsForward() const { return isForward; }
+    bool IsFFp8() const
+    {
+        const auto& fptr = GetDescsForward();
+        if(fptr.oDesc.GetType() == miopenFloat8)
+        {
+            return true;
+        }
+        else
+        {
+            return false;
+        }
```

Comment:
```suggestion
        return GetDescsForward().oDesc.GetType() == miopenFloat8;
```
But I'm not sure that it has to be here at all.

Reply:
I was following [convolution's problem description](https://github.com/ROCm/MIOpen/blob/develop/src/include/miopen/conv/problem_description.hpp#L314) pattern.

Reply:
Then my suggestion matches this pattern perfectly.